internal/pkg/template/templates/environment/partials/s3-elb-access-logs.yml

```diff
@@ -0,0 +1,39 @@
+ELBAccessLogsBucketPolicy:
+  Type: AWS::S3::BucketPolicy
+  Condition: CreateALB
+  DependsOn: ELBAccessLogsBucket
+  Properties:
+    Bucket: !Ref ELBAccessLogsBucket
+    PolicyDocument:
+      Version: '2012-10-17'
+      Statement:
+        - Action:
+            - s3:PutObject
+          Effect: Allow
+          Resource:
+            - !Join
+              - ''
+              - - 'arn:'
+                - !Ref AWS::Partition
+                - ':s3:::'
+                - !Ref ELBAccessLogsBucket
+                - '/'
+                - {{ .BucketPrefix }}
+                - '/AWSLogs/'
```

Comment:
If the `.BucketPrefix` is empty, I think this will result in `//AWSLogs/`  should this be something like:
```
{{- if .BucketPrefix }}
- '/'
- {{.BucketPrefix }}
{{- end}}
- '/AWSLogs/'

```

Reply:
We can do as you mentioned but in our implementation there won't be case where bucket prefix is not defined so i kept it this way but happy to change it.